Oracle/shrine_AdapterMappings.sql

```diff
@@ -14,36 +14,74 @@ drop table SHRINE_ONT.temp_shrine_mapping;
 CREATE TABLE SHRINE_ONT.temp_shrine_mapping
    AS (
        -- diagnosis icd9
-        select '\\SHRINE'|| so.c_fullname scihls_path,
-               '\\i2b2_Diagnoses' || ht.c_fullname heron_path 
+        select
+        	'\\SHRINE'|| so.c_fullname scihls_path
+            ,'\\i2b2_Diagnoses' || ht.c_fullname heron_path 
         from SHRINE_ONT.shrine so
         join blueheronmetadata.heron_terms ht
           on so.c_basecode=ht.c_basecode
           and so.c_basecode like 'ICD9:%' 
           and ht.C_FULLNAME like '\i2b2\Diagnoses\%' 
         union ALL
         -- diagnosis icd10
-        select '\\SHRINE'|| so.c_fullname scihls_path,
-               '\\i2b2_Diagnoses' || ht.c_fullname heron_path 
+        select
+        	'\\SHRINE'|| so.c_fullname scihls_path
+            ,'\\i2b2_Diagnoses' || ht.c_fullname heron_path 
         from SHRINE_ONT.shrine so
         join blueheronmetadata.heron_terms ht
           on so.c_basecode=ht.c_basecode
           and so.c_basecode like 'ICD10:%' 
           and ht.C_FULLNAME like '\i2b2\Diagnoses\%'
         union ALL
         -- demographics 
-        select '\\SHRINE'|| so.c_fullname scihls_path,
-               '\\i2b2_Demographics' || ht.c_fullname heron_path 
+        select
+        	'\\SHRINE'|| so.c_fullname scihls_path
+            ,'\\i2b2_Demographics' || ht.c_fullname heron_path 
         from SHRINE_ONT.shrine so
         join blueheronmetadata.heron_terms ht
           on so.c_basecode=ht.c_basecode
           and so.c_basecode like 'DEM%' 
           and ht.C_FULLNAME like '\i2b2\Demographics\%'
         union ALL
         -- NAACCR (tumor registry)
-        select '\\SHRINE_NAACCR' || c_fullname as scihls_path,
-               '\\i2b2_naaccr' ||  c_fullname as heron_path
+        select 
+        	'\\SHRINE_NAACCR' || c_fullname as scihls_path
+            ,'\\i2b2_naaccr' ||  c_fullname as heron_path
         from SHRINE_ONT.NAACCR_ONTOLOGY
+        union ALL
+		-- CPT Procedure Mapping
```

Comment:
why treat CPT different from ICD10? Doesn't the whole procedure ontology fit the same pattern?

Reply:
I can combine ICD9 and CPT.
But ICD10 needs to be separate as I cant simply do join on c_basecode. since heron c_basecode is starting with `ICD10:` when shrine c_basecode is starting with `ICD10PCS:`

On second thought, It might be possible for all (ICD9, ICD10 and CPT) if all codes are unique without the corresponding prefix (eg `CPT:`, `ICD9:`).

Reply:
I don't see how codes are relevant. The codes don't appear in the mapping, do they?

Reply:
They do not appear, but I am using them to generate mapping file.